src/plus/integrations/providers/models.ts

```diff
@@ -235,6 +237,14 @@ export type GetPullRequestsForAzureProjectsFn = (
 	options?: EnterpriseOptions,
 ) => Promise<{ data: ProviderPullRequest[] }>;
 
+export type MergePullRequestFn = (
+	input: {
+		pullRequest: { headRef: { oid: string | null } | null } & SetPullRequestInput;
+		mergeStrategy?: GitMergeStrategy.Squash;
```

Comment:
Because it's defined in the API: https://github.com/gitkraken/provider-apis-package-js/blob/0c56e4935e3490f89305c78fff5607694e6fbbe9/src/providers/gitlab/gitlab.ts#L1532-L1550

Reply:
Fixed